app/models/maintenance_tasks/task.rb

```diff
@@ -208,6 +226,14 @@ def load_constants
 
         Rails.autoloaders.main.eager_load_namespace(namespace)
       end
+
+      def default_handler(error)
+        if Rails::VERSION::MAJOR >= 7
```

Comment:
I still think we should just drop support below 7, but this won't affect many people so 🤷